src/SearchResults.js

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import moment from "moment";
+
+const SearchResults = props => {
+  //created to calculated the length of stay, any idea for the improvments??
+
+  // const checkTheLengthOfTheStay = props.results.map((element) => {
+  //   const a = moment(element.checkInDate);
+  //   const b = moment(element.checkOutDate);
+  //   return a.diff(b, "days") * -1;
+  // });
+
+  return (
+    <table className="table table-sm">
+      <thead>
+        <tr>
+          <th scope="col">Title</th>
+          <th scope="col">First Name</th>
+          <th scope="col">Surname</th>
+          <th scope="col">Email</th>
+          <th scope="col">Room id</th>
+          <th scope="col">Check in date</th>
+          <th scope="col">Check out date</th>
+          <th scope="col">Booking length</th>
+        </tr>
+      </thead>
+      <tbody>
+        {props.results.map((element, index) => (
```

Comment:
Could you use a better name here rather than `element`? Maybe `booking`?

This might seem like small issue now but when you're working on lots of code it makes it much easier :)

Reply:
Correction done
Thanks Chris